src/BugTracker/Jira.php

```diff
@@ -259,13 +248,4 @@ private function formatComment($timelog)
             )
         );
     }
-
-    /**
-     * Preparing this for JIRA, e.g. removing #hashmark, transforming "#scl-123" to "SCL-123"
-     */
-    public function sanitizeTicketId($ticketId)
-    {
-        $ticketId = trim(strtoupper(str_replace("#", "", $ticketId)));
```

Comment:
By removing this function entirely it also seem as if we drop the call to `strtoupper()`. Will that cause problems?

Reply:
`extractIds()` already uppers the ticket IDs. As far as I could tell, the only place it was used, it was used on a string gotten from `extractIds()`.